src/app/entry/state/current-collections.query.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+import { QueryEntity } from '@datorama/akita';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { CollectionOrganization } from '../../shared/swagger';
+import { CurrentCollectionsState, CurrentCollectionsStore } from './current-collections.store';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CurrentCollectionsQuery extends QueryEntity<CurrentCollectionsState, CollectionOrganization> {
+  currentCollectionOrganizations$ = this.selectAll().pipe(map((collectionOrganizations: Array<CollectionOrganization>) => {
+    collectionOrganizations.sort((a, b) => this.sortCollectionOrganizations(a, b));
+    return collectionOrganizations;
+  }));
+  currentCollectionIds$: Observable<Array<number>> = this.currentCollectionOrganizations$.pipe(
+    map(collectionOrganizations => collectionOrganizations.map(collectionOrganization => collectionOrganization.collectionId)));
+  constructor(protected store: CurrentCollectionsStore) {
+    super(store);
+  }
+
+  sortCollectionOrganizations(a: CollectionOrganization, b: CollectionOrganization): number {
```

Comment:
Is organizationName all lower or upper case? If no, then this should be case-insensitive sort.